### DISCUSSION TITLE
Tanix TX3 Mini Plus

### DISCUSSION THREAD

**Discussion by ghost** [devmfc/debian-on-amlogic#60] [Q&A]
I bought TX3 Mini Plus from Lazada 2 years ago, and it has an S905W2 chip, I flashed debian image on an sdcard, used Tanix W2 DTB and it works
![Untitled](https://github.com/devmfc/debian-on-amlogic/assets/164004041/7c182404-36b5-43e1-b804-3ed548a99209)
Boots, HDMI doesn't work with only framebuffer console works, wifi didn't work, same as Tanix W2
![Untitled2](https://github.com/devmfc/debian-on-amlogic/assets/164004041/5414c721-8dc4-444a-9b95-f72d307d48d6)

**Comment by kirkng** [comment]
Your post has given me a new idea. I have just purchased a TX3 Mini Plus 4GB/ 32GB and tried to install a Linux system. Here are my attempts and findings:

I tested Armbian, but it didn't boot from the TF card.
![tfcard boot err r_20240522082116](https://github.com/devmfc/debian-on-amlogic/assets/5362294/540f90b5-4028-42c4-84ef-cb535d4a4934)
I tried Debian-on-Amlogic, and it wouldn't boot normally under the s905w2_generic parameter. It required adding the parameters "bootargs0=pd_ignore_unused clk_ignore_unused" to boot successfully, but then an error message "mmc0: error-110 whilst initialising sd card" appeared.
Using the tanixw2 parameter resulted in the same "mmc0: error-110 whilst initialising sd card" error message.
Seeing your pictures, I tested using the USB port and found that it can boot and write to the eMMC normally.
![cardrader fromusb 20240524155638](https://github.com/devmfc/debian-on-amlogic/assets/5362294/529bd6ba-d83c-4f66-b241-22cf30a5cdd9)
![cardrader fromusb 202405241556382](https://github.com/devmfc/debian-on-amlogic/assets/5362294/ee8bae78-e059-4793-88f5-59412de0a653)
![cardrader fromusb 20240524155634](https://github.com/devmfc/debian-on-amlogic/assets/5362294/20674001-a6da-4ee7-963f-f72fb91c00ec)
![cardrader fromusb 20240524155636](https://github.com/devmfc/debian-on-amlogic/assets/5362294/1396204c-fc22-45ea-a6ac-160aa2df50a5)

My next steps will be to continue testing if I can install Python and further test Armbian.
![cardrader fromusb installemmc 20240524155637](https://github.com/devmfc/debian-on-amlogic/assets/5362294/8270778d-febb-49f6-85da-e0eaa46709d8)
![tx3mini board 20240524155718](https://github.com/devmfc/debian-on-amlogic/assets/5362294/c2c7d8db-beba-4df1-94e2-310eeace4690)
![tx3mini board 202405241557181](https://github.com/devmfc/debian-on-amlogic/assets/5362294/be0f5120-0091-417d-bb66-9cb2edebe9eb)
![tx3mini board 202405241557182](https://github.com/devmfc/debian-on-amlogic/assets/5362294/097f7053-6538-41e0-bd9b-143d16ee3034)
![tx3mini board 20240524155719](https://github.com/devmfc/debian-on-amlogic/assets/5362294/9df3d538-6b31-4c8c-91b0-6b5feda63ae9)
![tx3mini board 202405241557183](https://github.com/devmfc/debian-on-amlogic/assets/5362294/d12be91c-a44c-4a6c-aaf6-d2ef6f340506)
![tx3mini board 202405241557191](https://github.com/devmfc/debian-on-amlogic/assets/5362294/22348ccb-e21f-49d2-a17d-f34e10d0fc79)
![tx3mini board 202405241557192](https://github.com/devmfc/debian-on-amlogic/assets/5362294/cebf67f6-dfc9-4cec-8a31-51459355905f)
![tx3mini board 202405241557193](https://github.com/devmfc/debian-on-amlogic/assets/5362294/a33708f4-4792-432c-a96d-e1d7ab5f79d9)
![bootloader 202405241556391](https://github.com/devmfc/debian-on-amlogic/assets/5362294/7f80a129-e681-424e-bf94-9eb26e8ae414)

**Comment by ghost** [comment]
> I tested Armbian, but it didn't boot from the TF card.

Armbian does not provide S905W2 image

> I tried Debian-on-Amlogic, and it wouldn't boot normally under the s905w2_generic parameter. It required adding the parameters "bootargs0=pd_ignore_unused clk_ignore_unused" to boot successfully, but then an error message "mmc0: error-110 whilst initialising sd card" appeared.
Using the tanixw2 parameter resulted in the same "mmc0: error-110 whilst initialising sd card" error message.

A2 sd card won't work and it won't boot with these
Works fine on A1 sd card on an TX3 Mini Plus 2gb ram
